go.sum

```diff
@@ -0,0 +1,2 @@
+github.com/jschauma/getpass v0.2.1 h1:ivV/I5L7LHl81V5RdI0Mz1evFLU+bvpV03ENVI032Bc=
+github.com/jschauma/getpass v0.2.1/go.mod h1:mk40NJ7cAbcRkeDKDIajUsfQB/yweyiqjf29kyMnS18=
```

Comment:
remove this entire file since its not needed anymore.

Reply:
done